account_analytic_required/tests/test_account_analytic_required.py

```diff
@@ -1,85 +1,86 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Account analytic required module for OpenERP
-#    Copyright (C) 2014 Acsone (http://acsone.eu). All Rights Reserved
-#    @author Stéphane Bidoul <stephane.bidoul@acsone.eu>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
+# -*- coding: utf-8 -*-
+# © 2014 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 from datetime import datetime
 
 from openerp.tests import common
-from openerp.osv import orm
+from openerp import exceptions
 
 
-class test_account_analytic_required(common.TransactionCase):
+class TestAccountAnalyticRequired(common.TransactionCase):
 
     def setUp(self):
         super(test_account_analytic_required, self).setUp()
```

Comment:
s/test_account_analytic_required/TestAccountAnalyticRequired/

So Travis is not happy.